### PR TITLE
Ads credit success modal

### LINKED ISSUE #566: Ads Credit Post onboarding modal when feature is disabled.
When user finishes the onboarding wizard and the ads credit feature is not present he should be greeted with the following modal:

image

### PR DESCRIPTION
**Pull request by budzanowski**
### Changes proposed in this Pull Request:

Closes #566 .

Just adding the modal.

### Screenshots:

image

### Detailed test instructions:

1. Use this branch
2. Onboard
3. Observe the modal

### Changelog entry

>

### CODE REVIEW

File: assets/source/catalog-sync/components/OnboardingSuccessModal.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+			onRequestClose={ onCloseModal }
+			className="pinterest-for-woocommerce-catalog-sync__onboarding-success-modal"
+		>
+			<Text variant="title.small">
+				{ __(
+					'You have successfully set up your Pinterest integration.',
+					'pinterest-for-woocommerce'
+				) }
+			</Text>
+			<Text variant="body.large">
```
Comment (alexiglesias31) [inline assets/source/catalog-sync/components/OnboardingSuccessModal.js:41]: If the designs are  flexible, I'd aim for using the same body style on both modals (either `body` or `body.large`) in order to get consistency.

**Review by alexiglesias31 — APPROVED** [review]
LGTM!
Just a minor comment :wink: 
Also, if you don't mind, may I push (or by yourself) a commit renaming the component `AdsOnboardingModal` to `OnboardingModal` to get some consistency too :slightly_smiling_face:

File: assets/source/catalog-sync/components/OnboardingSuccessModal.js (hunk 1 above)
Comment (budzanowski) [reply assets/source/catalog-sync/components/OnboardingSuccessModal.js:41]: I think that we should go with `body.large` as it renders as:

```css
font-weight: 400;
font-size: 16px;
line-height: 24px;
```

which is basically what I see in the Figma design.

Comment (budzanowski) [reply assets/source/catalog-sync/components/OnboardingSuccessModal.js:41]: We will for sure do UI tweaks later next week during polishing touches.

Comment (budzanowski) [reply assets/source/catalog-sync/components/OnboardingSuccessModal.js:41]: > Also, if you don't mind, may I push (or by yourself) a commit renaming the component AdsOnboardingModal to OnboardingModal to get some consistency too 🙂

Done.

**Pull request merged**